src/components/shared/editable-text-input-list/editable-text-input-list.tsx

```diff
@@ -3,37 +3,65 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Button, Form, InputGroup } from "react-bootstrap";
+import { Button, Form, InputGroup } from 'react-bootstrap';
+import PageBlock from '../page-block/page-block';
 
 interface EditableTextInputListProps {
+  title: string;
   items: any[];
+  readOnly?: boolean;
+  ordered?: boolean;
   add: (val: any) => any;
   remove: (idx: number) => any;
   update: (idx: number, val: any) => any;
 }
 
 const EditableTextInputList: React.FC<EditableTextInputListProps> = ({
+  title,
   items,
+  readOnly,
+  ordered,
   add,
   remove,
   update
 }) => {
+  let listPrepared = items.map((item, index: number) =>
+    !readOnly ? (
+      <div key={index} className={'mb-2'}>
+        <InputGroup>
+          <Form.Control
+            type="text"
+            value={item.toString()}
+            onChange={(e) => {
+              update(index, e.target.value);
+            }}
+          />
+          <Button type="button" variant="danger" onClick={() => remove(index)}>
+            X
+          </Button>
+        </InputGroup>
+      </div>
+    ) : (
+      <li key={index}>{item}</li>
+    )
+  );
+
+  const addButton = (
+    <Button type="button" variant="success" onClick={() => add('')}>
+      + Add New Bullet
+    </Button>
+  );
+
+  if (!readOnly) {
+    listPrepared = [...listPrepared, addButton];
+  }
+  let builtList = <ul>{listPrepared}</ul>;
+  if (ordered) {
+    builtList = <ol>{listPrepared}</ol>;
+  }
```

Comment:
I don't think we need nor want to have HTML list tags in this stuff. If it isn't in there for the project edit form then let's not add it for this.

It looks like unnecessary indentation to me
<img width="818" alt="Screen Shot 2022-03-14 at 11 06 22 PM" src="https://user-images.githubusercontent.com/51571627/158297996-2eea0174-4635-4bf4-93c1-d0dbf287ab3c.png">

Reply:
I think it's fine if you want to turn this into a list HTML element, but I think a better way to go about it would be to replace the `div` element in the `map` with `li` and then wrap the output of the `map` in an `ul` so that it's something like `<ul>{mappedItems}</ul>`. Only thing with this is that you need to make sure to remove the default stylings from the list HTML elements (mainly `ul`, not sure if `li` also has default styling that needs to be reset)